lib/api/apiUtils/object/versioning.js

```diff
@@ -76,11 +76,18 @@ function _storeNullVersionMD(bucketName, objKey, objMD, options, log, cb) {
 * @param {string} objKey - name of object key
 * @param {object} options - metadata options for getting object MD
 * @param {string} options.versionId - version to get from metadata
+* @param {object} mst - info about the master version
```

Comment:
It's some versioning information extracted from the master version's object metadata, such as `versionId`, `isNull`, `isDeleteMarker` (see [here](https://github.com/scality/S3/pull/926/files/09b006811a5b2e03eb5549fdfd63f4d58228470f#diff-f595d5a19c0275ceb841fe9faa05e13eR193)). The only property we use in this function is `versionId` so that's the only one I included in the jsdoc.